personio-personnel-data-api-oa3.yaml

```diff
@@ -1267,6 +1277,186 @@ paths:
                       code: 404
                       message: 'The attendance period was not found.'
 
+  /company/attendances/projects:
+    parameters:
+      - $ref: "#/components/parameters/XPersonioPartnerID"
+      - $ref: "#/components/parameters/XPersonioAppID"
+    get:
+      security:
+        - BearerAuth: []
+      tags:
+        - Projects
+      description: Provides a list of all company projects.
+      responses:
+        "200":
+          description: List of projects returned
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  success:
+                    type: boolean
+                  data:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/Project"
+    post:
+      security:
+        - BearerAuth: []
+      tags:
+        - Projects
+      description: Creates a project into the company account
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - name
+              properties:
+                name:
+                  type: string
+                  example: "A project name"
+                active:
+                  type: boolean
+                  example: true
+                  default: false
+      responses:
+        "200":
+          description: Project created successfully
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  success:
+                    type: boolean
+                  data:
+                    $ref: "#/components/schemas/Project"
+        "400":
+          description: Invalid data was sent
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  success:
+                    type: boolean
+                    example: false
+                  error:
+                    type: object
+                    properties:
+                      name:
+                        type: array
+                        items:
+                          type: string
+                          example: "Please introduce a name to continue"
```

Comment:
.. to continue sounds a bit interactive here. `Project name is required` maybe? or `Project name cannot be empty`